torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1015,9 +1015,17 @@ at::Tensor& AtenXlaType::dropout_(at::Tensor& self, double p, bool train) {
 }
 
 at::Tensor AtenXlaType::einsum(std::string equation, at::TensorList tensors) {
-  if (tensors.size() != 2 || !ir::ops::Einsum::SupportsEquation(equation)) {
+  if (tensors.size() != 2) {
```

Comment:
What is this change?
Does not seem to matter with documentation.
I'd move to a different PR and explain better in the comment ...
Ditto for all the related C++ files in this PR.

Reply:
I couldn't build without this update. I'll remove it from this PR.

Reply:
Did you update TF?
I can see the signature could have changed on the TF side, but you should be using the commit ID which comes with our submodule.